etcddb/etcddb_test.go

```diff
@@ -281,7 +281,8 @@ func (suite *EtcdTestSuite) TestEtcdNilValue() {
 
 	ok, err = client.PutIfAbsent(key, firstValue)
 	assert.Nil(t, err)
-	assert.False(t, ok)
+	//If it was not Absent , we still can put the key successfully!
+	assert.True(t, ok)
```

Comment:
Not sure why you add this, but it should be replaced back
```suggestion
	assert.False(t, ok)
```